source/use-cases/home-resolver.rst

```diff
@@ -1,6 +1,128 @@
 Resolver for Home Networks
+==========================
+
+
+Setting up Unbound
+------------------
+
+Unbound is a powerful validating, recursive, caching DNS resolver. It’s used by some of the biggest tech companies in the world and also by home users who use it for Pi-hole or self-run resolvers. Setting it up for your home network can be quite simple as we’ll showcase below!
+
+Setting up your own DNS resolver for your entire home network requires a couple of things. Namely, a recursive DNS resolver (who knew!), and a dedicated machine where the resolver runs, which is always on and accessible to the entire network. This can be as simple as a Raspberry Pi connected to your home router or any other machine that you is always online!
+
+Because of the variaty of machines that Unbound can run on we cannot create a comprehensive tutorial for all possible options. For the extent of this tutorial we will use Ubuntu as a stepping stone which you could use for other machines. For this tutorial Ubuntu 20.04.1 LTS was used.
+
+While you could download the code and build it yourself if you wanted, it can be as simple as running:
+::
+	apt install unbound
+
+This gives you a full, compiled, and running version of Unbound which behaves as a caching recursive DNS resolver out of the box for the local machine.
+
+Do note that the current setup is only reachable on this machine.
+
+Testing the server locally
 --------------------------
 
-Using Unbound as the resolver for a home network.
+To verify that the server works correctly it’s a good idea to test it before committing the entire network to it. Luckily we can test this on the machine that you installed Unbound on (local) and from any other machine (remote)  that will be using the resolver after we expose Unbound to the network.
+
+The command for local testing is:
+::
+	dig example.com @127.0.0.1
+
+So we tell the dig tool to look up the IP address for example.com, and to ask this information to the server running at the ip address 127.0.0.1, which is where our Unbound machine is running.
+
+
+Setting up for a single machine
+-------------------------------
+
+Now that we have configured and tested our Unbound server, we can tell our machine to use it by default. We do this at:
+::
+	sudo nano /etc/systemd/resolved.conf
```

Comment:
Below you use wording such as "with your favorite text editor ...". I would also prefer that here instead of the example with nano. Nothing wrong with nano but if you are only used with GUI text editors that would be something,